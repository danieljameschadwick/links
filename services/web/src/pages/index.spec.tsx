import React from "react";
import { render } from "@testing-library/react-native";
import Index from "@src/pages/index";

describe("Index", () => {
  it("renders correctly", async () => {
    const { toJSON } = render(<Index />);

    expect(toJSON()).toMatchSnapshot();
  });
});
