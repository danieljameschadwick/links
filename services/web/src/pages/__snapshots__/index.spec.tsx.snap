// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<View>
  <Text>
    Links index.
  </Text>
</View>
`;
